INFO     logger:test_api.py:30 ————————case4————————
INFO     logger:test_api.py:44 ✅ 获取接口请求id：case4 
INFO     logger:test_api.py:49 ✅ 获取case4接口请求名称：用户名为空进行登陆 
INFO     logger:test_api.py:59 ✅ 获取case4接口请求url：http://39.98.138.157:5000/api/login 
INFO     logger:test_api.py:63 ✅ 获取case4接口请求类型：post 
INFO     logger:test_api.py:67 ✅ 获取case4接口请求头参数headers：None 
INFO     logger:test_api.py:71 ✅ 获取case4接口请求参数：{"username": "", "password": "123456"} 
INFO     logger:test_api.py:95 ✅ 获取case4接口请求响应结果：{'code': '001', 'msg': '用户名或密码错误'} 
INFO     logger:test_api.py:117 ✅ 获取case4接口请求响应字段'$..msg'的预期结果：用户名或密码错误 
INFO     logger:test_api.py:123 ✅ 获取case4接口请求响应字段'$..msg'的实际结果写入excel：用户名或密码错误 
INFO     logger:test_api.py:130 ✅ 获取case4接口请求响应断言成功，写入接口测试结果为'Pass'